### ISSUE TITLE
fix fetchData() or remove it from package for 1.0 release

### ISSUE THREAD

**Issue by rpruim** [ProjectMOSAIC/mosaic#364]
See various other issues (currently in "dormant issues") related to this that can be referenced.

**Comment by rpruim** [comment]
I'm planning to remove this from 1.0.    I've put in a deprecation notice for now, but I'll convert it to defunct for 1.0 even if we don't have another intermediate release.
 
If we want the functionality, it should move into a separate package (and receive the attention it needs to make it generally useful).  Note: the name `fetch` is available on CRAN.
```r
> fetchData("KidsFeet")
Data KidsFeet found in package.
 
<snip>

Warning message:
This function will be removed from the mosaic package in an upcoming release. 
```

**Comment by rpruim** [comment]
I've made `fetchData()` defunct in `mosaic` since it now lives in `fetch`.  Here's the message:
```r
mosaic::fetchData()
## Error: Use fetchData() from the fetch package instead.
```

**Comment by nicholasjhorton** [comment]
Randy,

I'm not seeing "fetchData()" as being deprecated in the current release of mosaic: therefore I feel that it's premature to have it disappear without any warning or notice.  

Can it be staged as "to be deprecated" in the news (with a possible warning)?

I see a number of uses of fetchData() and fetchGapminder() out in the wild based on some google searching.

Just my $0.02,

Nick

> On May 26, 2015, at 6:25 PM, Randall Pruim <notifications@github.com> wrote:
> 
> Closed #364.
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Nicholas Horton 
Professor of Statistics
Department of Mathematics and Statistics, Amherst College
Box 2239, 31 Quadrangle Dr
Amherst, MA 01002-5000
https://www.amherst.edu/people/facstaff/nhorton

**Comment by rpruim** [comment]
IF we do another pre-1.0 release, then I guess we could do this.  But I would not hold up 1.0 for a round of deprecation.

I don't see this as a big problem if the `fetch` package remains available with the functionality. 

Danny can make `fetch` a dependency of his other packages that support places where he uses `fetchData()` heavily.
